Common Example code/Networking/Environment.swift

```diff
@@ -14,9 +14,9 @@ internal enum DemoServerEnvironment: APIEnvironment {
     internal var baseURL: URL {
         switch self {
         case .beta:
-            return URL(string: "https://checkout-beta.adyen.com/v64")!
+            return URL(string: "https://checkout-beta.adyen.com/v100")!
         case .test:
-            return URL(string: "https://checkout-test.adyen.com/v64")!
```

Comment:
ApplePay is not working on version higher than v53. Probably not a good idea to pump version for now...

Reply:
Sorry, that's a [WIP] PR, but I will take this into consideration :-) .